.ci/docker/buildtools-windows/README.md

```diff
@@ -18,6 +18,16 @@ docker run --rm -ti \
        msbuild
 ```
 
+### If docker fails while downloading visual studio do the following
```

Comment:
A few lines above there is a section for the how to use it, that likely is deprecated:
- https://github.com/elastic/apm-agent-dotnet/blob/a81c4244c43cb9c4598d959a395b053289601397/.ci/docker/buildtools-windows/README.md#how-to-use-it